src/canvas-extractor/canvas_extractor/__main__.py

```diff
@@ -126,11 +121,8 @@ def _get_sections(sync_db: sqlalchemy.engine.base.Engine) -> None:
     logger.info("Extracting Sections from Canvas API")
     (courses, _) = results_store["courses"]
     (sections, udm_sections_df) = extract_sections(courses, sync_db)
-    write_csv(
-        udm_sections_df,
-        datetime.now(),
-        os.path.join(output_directory, SECTIONS_ROOT_DIRECTORY),
-    )
+    logger.info("Writing LMS UDM Sections to CSV file")
```

Comment:
What do you think about moving these log statements into the shared library? Is there any unwanted impact in terms of  the output logged to the screen/file?

Reply:
I don't have any strong opinion on favour or against this change.